runtime/src/main/kotlin/kotlin/native/concurrent/WorkerBoundReference.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package kotlin.native.concurrent
+
+import kotlin.native.internal.*
+
+@SymbolName("Kotlin_WorkerBoundReference_create")
+external private fun createWorkerBoundReference(value: Any): NativePtr
+
+@SymbolName("Kotlin_WorkerBoundReference_deref")
+external private fun derefWorkerBoundReference(ref: NativePtr): Any
+
+/**
+ * A frozen shared reference to a Kotlin object
+ *
+ * Can be safely passed between workers, but can only be accessed on the worker it was created on,
```

Comment:
> but can only be accessed on the worker

"It accessed" is ambiguous. Maybe explicitly mention `[value]` or "referred object", or use "dereference" instead of "access"?